pages/api/search/index.js

```diff
@@ -1,12 +1,12 @@
+import { searchUsers, searchUsersIfNotUsingAtlas } from "../profiles";
 import connectMongo from "@config/mongo";
-import { getProfiles } from "../profiles";
 import logger from "@config/logger";
 
 export default async function handler(req, res) {
   await connectMongo();
   const { slug } = req.query;
 
-  if (req.method != "GET") {
+  if (req.method !== "GET") {
```

Comment:
You can move methods to constants

Reply:
I haven't done this cause I worked with the mongo queries only. 
I think while pulled the changes to remove conflicts it somehow get added